### PR TITLE
Add my exer 14

### PR DESCRIPTION
**Pull request by Pasionpreciouskaye**
I add my exer 14

### CODE REVIEW

File: pasion_preciouskaye/countries.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Country Information Finder</h1>
+        <input type="text" id="country_Input" placeholder="Enter country name">
+        <button onclick="searchCountry()">Search</button>
+        <div id="country_details"></div>
+        <div id="same_regioncountries"></div>
+      </div>
```
Comment (pepito-alonajane) [inline pasion_preciouskaye/countries.html]: Observe proper indentation for your div tag

File: pasion_preciouskaye/scripts.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    commentsArray.forEach(comment => {
+      commentsList.appendChild(comment);
+    });
+  }
+});
+
+function searchCountry() {
+  var countryName = document.getElementById('countryInput').value.trim();
+  if (!countryName) {
+    document.getElementById('countryDetails').innerHTML = '<p>Please enter a country name.</p>';
```
Comment (pepito-alonajane) [inline pasion_preciouskaye/scripts.js:66]: Line exceeded 80 columns.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: pasion_preciouskaye/scripts.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    .then(function(countryData) {
+      var country = countryData[0];
+      var details = `
+        <h2>Country Details - ${country.name.common}</h2>
+        <img src="${country.flags.svg}" alt="Flag of ${country.name.common}" width="100">
+        <p><strong>Area:</strong> ${country.area ? country.area.toLocaleString() + ' square kilometers' : 'N/A'}</p>
+        <p><strong>Languages:</strong> ${country.languages ? Object.values(country.languages).join(', ') : 'N/A'}</p>
+        <p><strong>Subregion:</strong> ${country.subregion ? country.subregion : 'N/A'}</p>
+        <p><strong>Capital:</strong> ${country.capital ? country.capital[0] : 'N/A'}</p>
+        <p><strong>Timezones:</strong> ${country.timezones ? country.timezones.join(', ') : 'N/A'}</p>
```
Comment (pepito-alonajane) [inline pasion_preciouskaye/scripts.js]: Lines exceeded 80 columns
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: pasion_preciouskaye/scripts.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      if (!response.ok) {
+        throw new Error('Country not found');
+      }
+      return response.json();
+    })
+    .then(function(countryData) {
+      var country = countryData[0];
+      var details = `
+        <h2>Country Details - ${country.name.common}</h2>
+        <img src="${country.flags.svg}" alt="Flag of ${country.name.common}" width="100">
```
Comment (pepito-alonajane) [inline pasion_preciouskaye/scripts.js]: Please use external css styling
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g286f540102c_1_0

File: pasion_preciouskaye/scripts.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+        `;
+      }).join('');
+      document.getElementById('sameRegionCountries').innerHTML = `
+        <h2>Countries in the Same Region (${region})</h2>
+        <div>${sameRegionCountriesList}</div>
+      `;
+    })
+    .catch(function(error) {
+      console.error('Error fetching data:', error);
+      document.getElementById('countryDetails').innerHTML = '<p>An error occurred: ' + error.message + '</p>';
```
Comment (pepito-alonajane) [inline pasion_preciouskaye/scripts.js]: Line exceeded 80 columns.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

**Review by pepito-alonajane — COMMENTED** [review]
Please read my reviews. Thank you!

File: pasion_preciouskaye/countries.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Countries</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Country Information Finder</h1>
+        <input type="text" id="country_Input" placeholder="Enter country name">
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/countries.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
        <input type="text" id="country_input" placeholder="Enter country name">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: pasion_preciouskaye/countries.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        <button onclick="searchCountry()">Search</button>
+        <div id="country_details"></div>
+        <div id="same_regioncountries"></div>
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/countries.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
        <div id="same_region_countries"></div>
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: pasion_preciouskaye/index.html [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+          there are also some advantages. The disavantage students are often
+          lazy and when the camera is off they don't listen, sleep, or do 
+          anything else it is not a matter of the subject, because the 
+          interaction between the students and the instructor is less 
+          understandable and audible.
+        </p>
+    </div>
+
+    <div class="my-comment">
+      <form class="mycomment" id="comment" oninput="mycomment()">
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/index.html]: Class names should be written in kebab case. All lower case separated by hyphen. Also, Function names should be written in camel case. For reference, please click on the link provided below to access further information.


```suggestion
      <form class="my-comment" id="comment" oninput="myComment()">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?usp=sharing](url)

File: pasion_preciouskaye/index.html [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+      <h2>Contact Me</h2>
+      <form>
+        <div class="email">
+          <label type="email">Email</label>
+          <input type="email" id="email" name="email" />
+        </div>
+
+        <div class="fullname">
+          <label type="fullname">Full name</label>
+          <input type="text" id="fulllname" name="fullname" />
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/index.html]: It looks like you have a typo in your id. IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
          <input type="text" id="full_name" name="fullname" />
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: pasion_preciouskaye/index.html [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+          <label type="age">Age</label>
+          <input type="number" id="age" name="age" />
+        </div>
+
+        <div class="subject">
+          <label type="subject">Subject</label>
+          <input type="text" id="subject" name="subject" />
+        </div>
+
+        <div class="sendcopy">
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/index.html]: Class names should be written in kebab case. All lower case separated by hyphen. For reference, please click on the link provided below to access further information.


```suggestion
        <div class="send-copy">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

**Review by ElizaAlimasa — CHANGES_REQUESTED** [review]
Kindly review the suggested changes in your files and make sure to update your js and css file.

File: pasion_preciouskaye/scripts.js [hunk 11]
```diff
@@ -64,7 +64,7 @@ function searchCountry() {
   var countryName = document.getElementById('countryInput').value.trim();
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/scripts.js:64]: It looks like you forgot to update your js file. IDs should be written in snake case. All lower case separated by underscore. Also, always use "let" keyword in declaring variables over the "var" keyword. 

```suggestion
  let countryName = document.getElementById('country_input').value.trim();
```

For reference, please click on the link provided below to access further information. 
(https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5)

(https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0)

File: pasion_preciouskaye/index.html [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+
+    <div class="container">
+      <h2>Contact Me</h2>
+      <form>
+        <div class="email">
+          <label type="email">Email</label>
+          <input type="email" id="email" name="email" />
+        </div>
+
+        <div class="fullname">
```
Comment (ElizaAlimasa) [inline pasion_preciouskaye/index.html]: I forgot to mention this. Class names should be written in kebab case. All lower case separated by hyphen. For reference, please click on the link provided below to access further information.


```suggestion
        <div class="full-name">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5]

**Review by ElizaAlimasa — CHANGES_REQUESTED** [review]
Please review your code and update your css and js files base on the suggested changes

**Comment by Alimasa-Eliza** [comment]
Everything looks good to me.

**Review by pepito-alonajane — APPROVED** [review]
Your files are now ready for merging.

**Review by ElizaAlimasa — CHANGES_REQUESTED** [review]
Everything looks good to me. You can merge it to main now

**Review by ElizaAlimasa — APPROVED** [review]
Everything looks good to me. You can merge it to main now

**Comment by Pasionpreciouskaye** [comment]
Thankyou

**Pull request merged**